cmd/utils/flags.go

```diff
@@ -358,6 +359,12 @@ var (
 		Usage: "Disables price exemptions for locally submitted transactions" + generateEnvDoc(c_TXPoolPrefix+"nolocals"),
 	}
 
+	TxPoolSyncTxWithReturnFlag = Flag{
+		Name:  c_TXPoolPrefix + "sync-tx-with-return",
+		Value: true,
+		Usage: "Shares the tx with the sharing client with syncronous return" + generateEnvDoc(c_TXPoolPrefix+"sync-tx-with-return"),
```

Comment:
This also bypasses the tx pool right? You don't need to add it necessarily but might be useful info

Reply:
yeah, i will add that